code_samples/ai_actions/src/Command/AddMissingAltTextCommand.php

```diff
@@ -126,11 +127,13 @@ private function getSuggestedAltText(string $imageEncodedInBase64, string $langu
 
     private function convertImageToBase64(?string $uri): string
     {
-        $file = file_get_contents($this->projectDir . \DIRECTORY_SEPARATOR . 'public' . \DIRECTORY_SEPARATOR . $uri);
-        if ($file === false) {
-            throw new \RuntimeException('Cannot read file');
+        if ($uri === null) {
+            throw new \DomainException('Image field type is missing the uri property');
```

Comment:
I wouldn't mention 'Image field' here. You know it comes from an Image field only in `execute()`.